### DISCUSSION TITLE
Conditionally color rows except one special column?

### DISCUSSION THREAD

**Discussion by dushan-ranasinghage** [TanStack/table#2460] [General]
Is there a way to apply styling to a row except one or two column based on some condition? I tried out the below code snippet. But I couldn't find a way to access column details in `getTrProps`. 

`getTrProps={(state, rowInfo, column) => {
      console.log(state, rowInfo, column)
        return {
          style: {
            background: rowInfo && rowInfo.row.status == 0 ? '#FFFF00' : '#fff'
          }
        }
      }}`

This is the code which I am using to color rows according to the data I am getting from the back-end.

**Comment by tannerlinsley** [comment]
Feel free to use discord to ask questions as well: https://discord.com/invite/WrRKjPJ
